vshard/vshard-version.lua

```diff
@@ -0,0 +1,4 @@
+-- Сontains the module version.
+-- Requires manual update in case of release commit.
+
+return '0.1.22'
```

Comment:
Please, lets not add a whole new file just for a single constant. You can either hardcode it right in `vshard/init.lua`, or add it to `vshard/consts.lua`.

Reply:
Moved to `vshard/consts.lua`